### PR TITLE
D8CORE-2720 User card display mode for the views

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Updated config from stanford_profile
- Added styles for taxonomy term `<ul>` list elements.

# Need Review By (Date)
- 10/30

# Urgency
- medium

# Steps to Test
1. follow steps in https://github.com/SU-SWS/stanford_profile/pull/319

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: lib/scss/components/news-vertical-teaser/_news-vertical-teaser.scss [hunk 1]
```diff
@@ -164,6 +164,26 @@
             text-decoration: underline;
           }
         }
+
+        ul {
+          list-style: none;
+          margin: 0;
+          padding: 0;
```
Comment (sherakama) [inline lib/scss/components/news-vertical-teaser/_news-vertical-teaser.scss:171]:
```suggestion
          @include margin(0);
          @include padding(0);
```

File: lib/scss/components/news-vertical-teaser/_news-vertical-teaser.scss [hunk 2]
```diff
@@ -164,6 +164,26 @@
...
+        }
+
+        li {
+          display: inline-block;
+
+          &::after {
+            content: ',';
+            margin-right: 5px;
```
Comment (sherakama) [inline lib/scss/components/news-vertical-teaser/_news-vertical-teaser.scss:179]:
```suggestion
            @include margin(null 5px null null);
```

File: lib/scss/components/news-vertical-teaser/_news-vertical-teaser.scss [hunk 3]
```diff
@@ -164,6 +164,26 @@
...
+          }
+
+          &:last-child::after {
+            content: '';
+            margin-right: 0;
```
Comment (sherakama) [inline lib/scss/components/news-vertical-teaser/_news-vertical-teaser.scss:184]:
```suggestion
            @include margin(null 0 null null);
```

**Review by sherakama — APPROVED** [review]
GTG after sass changes.

**Comment by pookmish** [comment]
IMO it's really unnecessary to pass through all the mixins for something so minor. Its pretty tedious and results in the same outcome. But a topic for another day

**Pull request merged**